chat-android/src/test/java/com/ably/chat/SandboxTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.ably.chat
+
+import io.ably.lib.realtime.AblyRealtime
+import io.ably.lib.realtime.ChannelState
+import kotlinx.coroutines.test.runTest
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import io.ably.lib.types.ClientOptions as PubSubClientOptions
+
+class SandboxTest {
+
+    private val sandbox: Sandbox = Sandbox()
+
+    @Before
+    fun setUp() = runTest {
+        sandbox.setUp()
+    }
+
+    @After
+    fun tearDown() = runTest {
+        sandbox.tearDown()
+    }
+
+    @Test
+    fun basicIntegrationTest() = runTest {
+        val chatClient = createSandboxChatClient(sandbox.apiKey)
+        val room = chatClient.rooms.get("basketball")
```

Comment:
I'd definitely recommend the room id here (and anything you do in tests) random - if you're running an IDE that caches the sandbox config (or just running multiple tests in a suite), it gets flakey :D